trieste/utils/misc.py

```diff
@@ -210,3 +212,18 @@ def map_values(f: Callable[[U], V], mapping: Mapping[K, U]) -> Mapping[K, V]:
         applying ``f`` to each value in ``mapping``.
     """
     return {k: f(u) for k, u in mapping.items()}
+
+
+class timeit:
```

Comment:
There is a module with the same name in the standard lib: https://docs.python.org/3/library/timeit.html . I'd suggest to pick a different name to avoid confusions

Reply:
okay changed it to Timer